### ISSUE TITLE
Checkbox Accessibility Tab updates and DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-website#574]
This issue is related to #491
The following updates are needed to the Checkbox Accessibility Tab:
**1. Remove the first paragraph:** 
"The Checkbox React Carbon component has been tested against the latest W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria and violations have been identified as high priority issues. This document will be updated when these accessibility issues are resolved."

**Note:** This paragraph is part of the Non-compliant template and should not be added to the completed documents. The non-compliant template is used when the document can not be completed due to open accessibility issues that block completion.

**2. Typically a different font is used for navigation keys.** Is there a Carbon font that can be used? If there is, the following navigation keys should be updated in the second paragraph under how it works:
Tab 
Space 

**3. Change the font to represent code for aria mark-up ** in the second paragraph under how it works:**
aria-checked="true" 
aria-checked="false" 
aria-checked="mixed" 

**4. Under the Heading "Accessibility Testing" add the following paragraph:**
Accessibility issues are tracked in the [Carbon Component GitHub repository](https://github.com/carbon-design-system/carbon/milestone/57).

**5. Update Automated Test Environment > Results because new DAP violations were reported.  Change to: "DAP Test - Violations"** 

**6. The following DAP violations are also found. Let me know if they need to be reported on a separate issue (see screenshot below):**
<img width="1235" alt="Screen Shot 2019-11-11 at 2 23 56 PM" src="https://user-images.githubusercontent.com/21676914/68619163-8320fb80-0490-11ea-87eb-3f3d221096dc.png">

**Comment by aledavila** [comment]
@snidersd could you report the DAP violations in a separate issue under this repo. Let's keep this issue to track docs.

**Comment by snidersd** [comment]
@aledavila @dakahn - Can you confirm with DA. We discussed the doc issues this morning and he advised me to keep the issues together. Also note, there are DAP violations specific to the documentation. Thx
